packages/crud/test/crud-buttons.test.js

```diff
@@ -570,4 +599,33 @@ describe('crud buttons', () => {
       });
     });
   });
+
+  describe('lazy', () => {
+    beforeEach(async () => {
+      crud = fixtureSync('<vaadin-crud style="width: 300px;"></vaadin-crud>');
+      await nextRender(crud._grid);
+      [saveButton, cancelButton, deleteButton] = crud.querySelectorAll('vaadin-button');
```

Comment:
This seems a bit brittle as it relies on the DOM structure - is there an alternative way to identify the individual buttons? By slot name maybe?

Reply:
👍  Done.